examples/cis2-multi/src/lib.rs

```diff
@@ -524,7 +524,8 @@ impl State {
 #[init(
     contract = "cis2_multi",
     event = "Cis2Event<ContractTokenId, ContractTokenAmount>",
-    parameter = "TokenAmountU64"
+    parameter = "TokenAmountU64",
+    error = "ContractError"
```

Comment:
This is not correct I think. Unless I'm mistaken `InitResult<S> = Result<S, Reject>` defined in concordium-std.